mage/algorithms/dynamic-graph-analytics/betweenness-centrality-online-algorithm.md

```diff
@@ -0,0 +1,80 @@
+
+> After the node in the center is erased, shortest paths run through other nodes and their betweenness centrality scores
+> increase. Larger node size signifies higher betweenness centrality score.
+
+## Materials
+
+### Implementation
+
+[![Betweenness
+Centrality Online](https://img.shields.io/badge/Betweenness_Centrality-Online-Implementation-FB6E00?style=for-the-badge&logo=github&logoColor=white)](https://github.com/memgraph/mage/blob/main/cpp/betweenness_centrality_module/betweenness_centrality_module.cpp)
+
+[![Betweenness
+Centrality Online](https://img.shields.io/badge/Betweenness_Centrality-Online-Documentation-FCC624?style=for-the-badge&logo=cplusplus&logoColor=white)](/mage/query-modules/cpp/betweenness-centrality-online)
+
+Dynamic Betweenness Centrality is implemented as part of the
+[**MAGE**](https://github.com/memgraph/mage) project. Be sure to check it out at
+the link above. :point_up:
+
+### Blog posts
+
+(to be written)
+
+## Use cases
+
+Real-world networks are often dynamic and evolve rapidly. Consequently, re-running static algorithms 
+after each update often proves infeasible. Dynamic algorithms avoid redundant work by processing only 
+the portion of the network affected by the latest update.
+
+[![Social
+networks](https://img.shields.io/badge/Social_networks-Application-8A477F?style=for-the-badge)](/use-cases/social-media.md)
+
+**Social networks** are a quintessential example of rapidly evolving graphs – people step in and out
+of contact every day, and even a single change can significantly alter the network of relationships.
+Furthermore, individual users’ activities often come in *bursts* of many interactions in short order.
+With real-time network-aware applications (e.g. finding influential persons, trend analysis), being able to
+instantly reflect graph changes into centrality values is crucial.
+
+[![Logistics](https://img.shields.io/badge/Logistics-Application-8A477F?style=for-the-badge)](/use-cases/transportation.md)
+
+Another key use case of this algorithm is **logistics**/**transportation**. In 
+[complex transportation networks](https://www.hindawi.com/journals/jat/2019/9024745/),
+measuring betweenness centrality helps identify bottlenecks and choke points, as well as monitor
+how traffic is redistributed as nodes or connections between them become open or close.
+This can help with applications such as shipping/delivery routes or urban traffic grid optimization
+on both cost and benefit fronts.
+
+[![Bioinformatics](https://img.shields.io/badge/Bioinformatics-Application-8A477F?style=for-the-badge)](/use-cases/bioinformatics.md)
+
+[**Protein-protein interaction
+networks**](https://www.ebi.ac.uk/training/online/courses/network-analysis-of-protein-interaction-data-an-introduction/protein-protein-interaction-networks/)
+were created to get a better insight into how proteins interact with each other.
```

Comment:
Are dynamic protein-protein interaction networks used in practice? If not, I'm ready to remove this.

Reply:
I think they are used. We got a hint from one developer that is using Memgraph.

Reply:
Thanks! Could you share what they said with me?